.github/workflows/test_bundle.yml

```diff
@@ -0,0 +1,75 @@
+name: colcon-bundle Integration Test
+on:
+  pull_request:
+  schedule:
+    - cron: '0 * * * *'
+
+# Test the released and current master branch
```

Comment:
Do we need to test released and current master branch for all pull requests?

Reply:
Yes: if a contribution changes the vcs repos file (colcon-bundle/.github/workflows/test_bundle.repos) we won't catch breaking changes until merged.